### DISCUSSION TITLE
【屏蔽黑名单up主】拥堵，急需优化

### DISCUSSION THREAD

**Discussion by homelie** [the1812/Bilibili-Evolved#4749] [功能建议 / Ideas]
大家好，B站真是让人又爱又恨，有许多宝藏视频等待发掘，又有许多在作死边缘的擦边视频，更多的是一些没营养的歪曲解读的标题党视频，【屏蔽黑名单up主】功能真的是及时雨，可是黑名单太多，把UI都占满了，有其他人有同样的经历吗？望作者大大优化优化，感谢哈🙏

![截屏2024-05-17 11 50 29](https://github.com/the1812/Bilibili-Evolved/assets/2018666/931f09f2-5684-449b-bc86-66ad384a88bb)

这样不算夸张吧，500个B站黑名单也满了，还是不够呀😜

**Comment by homelie** [comment]
优化小小建议，UI被名单们占满了，无法显示名单的入口了，可否将添加入口与名单列表分开（逻辑/业务功能独立）展示，感谢抽空查看哈。

**Comment by the1812** [comment]
记得 at 下作者: @snowraincloud

**Comment by snowraincloud** [comment]
@homelie 我这修复下，加下下拉框。

**Comment by snowraincloud** [comment]
@homelie @the1812 已提pr修复 https://github.com/the1812/Bilibili-Evolved/pull/4784
